game_api.txt

```diff
@@ -155,7 +155,7 @@ The doors mod allows modders to register custom doors and trapdoors.
 ### Fence gate definition
 
 	description = "Wooden Fence Gate",
-	texture = "default_wood.png",
+	texture = "default_wood.png", -- `backface_culling` will automatically be set to `true` if it's not given.
```

Comment:
Sorry, just realised that, and misread what you wrote.
We still try to limit docs to 80 columns, again to be readable when splitting a screen.